tests/benchmark/dags/evaluate_load_file.py

```diff
@@ -47,6 +47,7 @@ def create_dag(database_name, table_args, dataset):
 
     with DAG(dag_name, schedule_interval=None, start_date=START_DATE) as dag:
         chunk_size = int(os.getenv("ASTRO_CHUNK_SIZE", str(DEFAULT_CHUNK_SIZE)))
+        use_native_support = bool(int(os.getenv("ASTRO_ENABLE_NATIVE_LOAD", "1")))
```

Comment:
@tatiana do you think we can move this feature to `config.json` `datasets` section? and that way we can run even the comparisons with native/non native paths?